lib/react_on_rails/configuration.rb

```diff
@@ -266,6 +254,32 @@ def raise_missing_components_subdirectory
 
       raise ReactOnRails::Error, msg
     end
+
+    def shakapacker_precompile?
+      return Webpacker.config.webpacker_precompile? if ReactOnRails::WebpackerUtils.using_shakapacker_6?
+
+      Webpacker.config.shakapacker_precompile?
```

Comment:
I think we should do this when we drop support for Webpacker/Shakapacker 6.

This way, we still know we are supporting the old interface. Otherwise, changing the name is not a big issue and can be done in a separate PR.